src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -251,6 +253,19 @@ export default Vue.extend({
       }
     },
 
+    determineMaxFramerate: function() {
+      if (this.dashSrc[0] !== undefined) {
+        fs.readFile(this.dashSrc[0].url, (err, data) => {
+          if (err) throw err
```

Comment:
Since this will always error when the Invidious API is selected, we probably just don't want to throw an error into the console.  There's nothing catching this on the other side as well, so throwing doesn't help us too much in this situation and can prevent code execution depending on where `determineMaxFramerate` is called in the future.

We'll want to change this to do a couple of things. First, we'll let it default the `maxFramerate` to 60 (as per our discussion). Next, we'll want to `return` so that the next statement block isn't executed, since at this point, `data` isn't defined.

We can still log this error if we want with `console.log()` or with `console.error()` if we feel like seeing this error needs to be a little more visible, though a `console.log` or simply not logging anything is fine, since this would error out on every video if Invidious is the default.